functions/public/Invoke-WPFShortcut.ps1

```diff
@@ -27,8 +27,10 @@ function Invoke-WPFShortcut {
             $ArgumentsToSourceExe = "$powershell '$IRM'"
             $DestinationName = "WinUtil.lnk"
 
+            Invoke-WebRequest -Uri "https://christitus.com/images/logo-full.png" -OutFile "$env:TEMP\cttlogo.png"
+
             if (Test-Path -Path "$env:TEMP\cttlogo.png") {
-                $iconPath = "$env:SystempRoot\cttlogo.ico"
+                $iconPath = "$env:TEMP\cttlogo.ico"
```

Comment:
The use of temp could be problematic in the future. When Windows eventually clears the temp folder, the file will vanish and the generated link could lose the icon again
I would probably opt for creating a folder "winutil" or something under appdata\local

Reply:
I do agree on this, not to mention that the Window showing delay _is actually_ because of the `<Image />` Element in `inputXML.xaml` File, which will reject handling any User Input, until this file is downloaded, so after working on it (did some experiments here and there), it works as expected.

For this PR, it'll be only these changes.

Reply:
Also, thanks for the suggestion @Marterich 😄